LeaderboardBackend/Models/Entities/ApplicationContext.cs

```diff
@@ -31,9 +32,22 @@ public ApplicationContext(DbContextOptions<ApplicationContext> options)
     public void MigrateDatabase()
     {
         Database.Migrate();
+        bool tempConnection = false;
+        NpgsqlConnection connection = (NpgsqlConnection)Database.GetDbConnection();
```

Comment:
When does the connection drop?

Reply:
I don't understand the question. Can you rephrase?